background-charm-service/src/worker.ts

```diff
@@ -64,17 +105,17 @@ async function initialize(
   // Initialize charm manager
   manager = new CharmManager(currentSession);
 
-  console.log(`Worker: ${did} initialized`);
+  console.log(`Initialized`);
   initialized = true;
 }
 
 // FIXME(ja) should we make sure we kill the worker?
 async function cleanup(): Promise<void> {
   if (!initialized) {
-    console.log(`Worker: Not initialized, skipping cleanup`);
+    console.log(`Not initialized, skipping cleanup`);
```

Comment:
missing did?

Reply:
or should trueConsole add that?

Reply:
https://github.com/commontoolsinc/labs/pull/1085/files#r2052990354